code/game/objects/items/weapons/implant/implant.dm

```diff
@@ -28,22 +28,25 @@
 	return external
 
 //return TRUE for implanter icon update.
-/obj/item/weapon/implant/proc/install(var/mob/living/carbon/human/target, var/organ, var/mob/user)
-	var/obj/item/organ/external/affected = target.organs_by_name[organ]
-	if(!affected)
-		if(allowed_organs.len)
-			organ = pick(allowed_organs)
-		else
-			organ = BP_CHEST
-	affected = target.organs_by_name[organ]
-
-	if(!affected)
-		user << SPAN_WARNING("[target] miss that body part!.")
-		return
-
-	if(allowed_organs && allowed_organs.len && !(organ in allowed_organs))
-		user << SPAN_WARNING("[src] cannot be implanted in this limb.")
-		return
+/obj/item/weapon/implant/proc/install(var/mob/living/target, var/organ, var/mob/user)
+	var/obj/item/organ/external/affected
+	if (ishuman(target))
+		var/mob/living/carbon/human/H = target
+		affected = H.organs_by_name[organ]
+		if(!affected)
+			if(allowed_organs.len)
+				organ = pick(allowed_organs)
+			else
+				organ = BP_CHEST
+		affected = H.organs_by_name[organ]
```

Comment:
Missed indent?

Reply:
No, that's indented exactly where it should be